doc/spec.md

```diff
@@ -258,8 +258,8 @@ var i = 0;
 TypeScript will infer from the following function definition that the function f has return type string.
 
 ```TypeScript
-function f() {  
```

Comment:
Editing the spec isn't as easy as that (it's complicated), so leave spec changes out of this PR.

Reply:
Done